template/android/fastlane/actions/appcenter_codepush.rb

```diff
@@ -0,0 +1,100 @@
+module Fastlane
```

Comment:
Why we need this file?

Reply:
because we need to create a function for FastLane, as you can see on Fastlane file, lane codepush